docs/conf.py

```diff
@@ -93,8 +94,42 @@ def check_if_v046():
 master_doc = 'index'
 
 # Multiversion settings
+# Show only the highest patch versions of each minor version.
+# Example: include 0.4.6, but not 0.4.0 to 0.4.5
+repo = Repo('.', search_parent_directories=True)
+all_tags = [tag.path.strip('refs/tags') for tag in repo.tags]
+all_tags = [tag for tag in all_tags if tag[0] == "v"]
+version_dict = {}
+for tag in all_tags:
+    split_tag = tag.split('.')
+    if len(split_tag) != 3:
+        print(f'ignoring version {tag}')
+        continue
+    major, minor, patch = split_tag
+    minor = int(minor)
+    patch = int(patch)
+
+    if major == 'v0' and minor < 4:
+        # ignore versions below 0.4
+        continue
+
+    if major in version_dict:
+        if minor in version_dict:
+            if patch > version_dict[major][minor]:
+                # enter new highest patch version
+                print(f'ignoring version {major}.{minor}.{version_dict[major][minor]}')
+                version_dict[major][minor] = patch
+        else:
+            # first time encountering this minor version
+            version_dict[major][minor] = patch
+    else:
+        version_dict[major] = {minor: patch}
```

Comment:
you probably want to use `packaging.version.parse` and friends here (https://stackoverflow.com/a/11887885/2536294)

Reply:
Switching to `packaging` in the next commit! Thanks for the suggestion.